common/persons_api/persons.go

```diff
@@ -0,0 +1,211 @@
+// To parse and unparse this JSON data, add this code to your project and do:
+//
+//    person, err := UnmarshalPerson(bytes)
+//    bytes, err = person.Marshal()
+
+package persons_api
+
+import "encoding/json"
+
+func UnmarshalPerson(data []byte) (Person, error) {
+	var r Person
+	err := json.Unmarshal(data, &r)
+	return r, err
+}
+
+func (r *Person) Marshal() ([]byte, error) {
+	return json.Marshal(r)
+}
+
+type Person struct {
+	AccessInformation AccessInformationValuesArray    `json:"access_information"`
```

Comment:
Is this struct fairly stable API wise? Will we need to keep this updated?

Reply:
It is still in beta, so I'd say we may need to update it.